### ISSUE TITLE
Make Quarto Multi-lingual or German only

### ISSUE THREAD

**Issue by mrchristian** [TIBHannover/semantic-glosar#14]
We need:

- Document language codes as German
- Labels as German - Inhalt, etw
- etc

Here are some resources:

https://docs.ropensci.org/babeldown/articles/quarto.html

This might be overkill - but lets see.

Please keep a record here of changes made in the system to either change to German or make multilingual.

Being multilingual would be best.

**Comment by calnfynn** [comment]
Quarto has its own ways of setting up multiple languages, but LaTeX has some kind of conflict with that. The `lang: de` bit loads the `babel` package, which breaks the entire thing because it's already being loaded by something else prior.     
Not sure if that is worth figuring out or too time-consuming. I'll do the styling first and then take another look. Alternatively we might be able to use a custom language file that lists all the labels and their translations.

**Comment by mrchristian** [comment]
Yep lets leave the multilingual parts on the side. I'll log your note as an issue its self. These can be things we go to the Quarto community with.

@calnfynn - does Quarto allow us to set the single language as DE?

**Comment by mrchristian** [comment]
DE changes:

If possible you should take updates from the Main Repo on https://github.com/TIBHannover/semantic-glosar/ to your AR repo - if possible.

The changes on Main /TIBHannover/semantic-glosar that are important are:

guide.md
cosite001.ipynb

DE Translation changes needed.

1. Change the left menu items to German - where needed
2. Translate page content on: About and Imprint
3. Note also change _quarto.yml title and author to: Semantic Glosar; Team Co-Site

When ready on AR ping me and once I check you can pull request to Main /TIBHannover/semantic-glosar/

**Comment by mrchristian** [comment]
Please go ahead with the lang: de changes
